chain-signatures/node/src/web/cbor.rs

```diff
@@ -0,0 +1,120 @@
+use axum::async_trait;
+use axum::body::Bytes;
+use axum::extract::{FromRequest, Request};
+use axum::http::StatusCode;
+use axum::response::{IntoResponse, Response};
+use http::{header, HeaderMap};
+use serde::{de::DeserializeOwned, Serialize};
+
+use crate::protocol::message::cbor_to_bytes;
+
+pub struct Cbor<T>(pub T);
```

Comment:
We may change or extend our messaging protocol. Should we add an int version to it now so we can handle it easier in the future?

Reply:
we shouldn't, as versioning should come with the serialization standard. cbor is like borsh where we have to handle it ourselves, but cbor does make it easier for future proofing since we can add in fields without making previous versions invalid. Adding a version number won't do anything in that case. Neither will it add much value for changing standards since we'd just reject on endpoint level anyways if it disagrees